### PR TITLE
LOCAL_DEBUG in MuonBaseNumber.cc restored

### PR DESCRIPTION
**Pull request by slomeo**
#### PR description:

Restored #ifdef LOCAL_DEBUG in MuonBaseNumber.cc as requested by @fabiocos

### CODE REVIEW

**Comment by slomeo** [comment]
@fabiocos : as you requested

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
Remember to update the title of this PR. It should fix the unit test in Geometry/DTGeometryBuilder (https://github.com/cms-sw/cmssw/pull/28796#discussion_r377507493)

**Comment by fabiocos** [comment]
@slomeo please update the PR title to something meaningful for the release notes

**Comment by silviodonato** [comment]
urgeny

**Comment by slomeo** [comment]
done

> @slomeo please update the PR title to something meaningful for the release notes

File: Geometry/MuonNumbering/src/MuonBaseNumber.cc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "Geometry/MuonNumbering/interface/MuonBaseNumber.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-//#define LOCAL_DEBUG
+#define LOCAL_DEBUG
```
Comment (cvuosalo) [inline Geometry/MuonNumbering/src/MuonBaseNumber.cc]: @slomeo Please explain this change. It turns on the debug statements. The problem in #28796 was that the debug statements were effectively turned on. I think you want to turn OFF the debug statements, which means this line should remain commented out. Am I missing something?

Comment (slomeo) [reply Geometry/MuonNumbering/src/MuonBaseNumber.cc]: @cvuosalo : The change is made to answer to @fabiocos  (see #28796)

Comment (cvuosalo) [reply Geometry/MuonNumbering/src/MuonBaseNumber.cc]: @slomeo But how does it answer Fabio's request? It seems to do the opposite of what he requested.

Comment (slomeo) [reply Geometry/MuonNumbering/src/MuonBaseNumber.cc]: @cvuosalo I could make a test: restore the version where only the line 4 is commented (i.e //#define LOCAL_DEBUG)

**Comment by silviodonato** [comment]
urgent

File: Geometry/MuonNumbering/src/MuonBaseNumber.cc (hunk 1 above)
Comment (silviodonato) [reply Geometry/MuonNumbering/src/MuonBaseNumber.cc]: Yes, as Fabio suggested, please make a version where only line 4 is commented.
It should appear like the left side of https://github.com/cms-sw/cmssw/pull/28796/files/a6175cdc9b547560bede5f274e12ca51e431d5dd#diff-47cb6cf59007852c622e5f745bab812dL4-L42

Comment (fabiocos) [reply Geometry/MuonNumbering/src/MuonBaseNumber.cc]: @slomeo I am afraid you misinterpreted my suggestion, we want to silence the messages but keep the possibility to switch them on, not to enable them by default

Comment (silviodonato) [inline Geometry/MuonNumbering/src/MuonBaseNumber.cc]:
```suggestion
//#define LOCAL_DEBUG
```

**Comment by silviodonato** [comment]
please test with 28875

**Comment by silviodonato** [comment]
ping @cmsbuild

**Comment by silviodonato** [comment]
please test with 28875

**Comment by silviodonato** [comment]
please test with #28875

**Comment by silviodonato** [comment]
merge
FYI geometry: @Dr15Jones @civanch @cvuosalo @ianna @makortel @mdhildreth

**Comment by cvuosalo** [comment]
+1

**Pull request merged**